app/test/e2e/utils/mocks.js

```diff
@@ -0,0 +1,437 @@
+                                                [
+                                                    -43.6015281677246,
+                                                    -2.2698609828949
+                                                ],
+                                                [
+                                                    -43.6218070983886,
+                                                    -2.27847099304199
+                                                ],
+                                                [
+                                                    -43.6404151916504,
+                                                    -2.27569508552551
+                                                ],
+                                                [
+                                                    -43.6548614501953,
+                                                    -2.29263806343079
+                                                ],
+                                                [
+                                                    -43.6662483215331,
+                                                    -2.30041694641113
+                                                ],
+                                                [
+                                                    -43.6665267944335,
+                                                    -2.3045830726623
+                                                ],
+                                                [
+                                                    -43.6725362197416,
+                                                    -2.30173635989672
+                                                ]
+                                            ],
+                                        ]
+                                    ],
+                                    type: 'MultiPolygon'
+                                },
+                                type: 'Feature',
+                                properties: null
+                            }
+                        ]
+                    },
+                    hash: 'a86236811c4ba13e0e70b8d94547e09d',
+                    provider: {},
+                    areaHa: 855189203.6641197,
+                    bbox: [
+                        -73.9882431030273,
+                        -33.7470817565917,
+                        -28.8472194671629,
+                        5.2648777961731
+                    ],
+                    lock: false,
+                    info: {
+                        use: {},
+                        iso: 'BRA',
+                        id1: null,
+                        id2: null,
+                        gadm: '2.8',
+                        name: 'Brazil'
+                    }
+                }
+            }
+        });
+};
+
+const mockTimeoutCartoDBRequest = () => {
+    const base = config.get('cartoDB.apiUrl').replace('/api/v2/sql', '');
```

Comment:
Couldn't we just build the url from the existing cartoDB.user config value?

Reply:
Not 100% sure if that was possible, hence why I did it this way, but using CartoDB user also works :+1: